docs/generating-a-provider.md

```diff
@@ -107,6 +114,8 @@ example organization name to be used.
    mkdir config/repository
    # Create custom configuration directory for whole branch group
    mkdir config/branch
+   # remove the sample directory which was a configuration in the template
+   rm -rf config/null
```

Comment:
Non-blocking question: Could we update the template so this step isn't necessary?

Reply:
I will check, and open an issue for it if possible.